programs/copier/ClusterCopier.h

```diff
@@ -187,8 +186,8 @@ class ClusterCopier
     UInt64 executeQueryOnCluster(
             const ClusterPtr & cluster,
             const String & query,
-            const ASTPtr & query_ast_ = nullptr,
-            const Settings * settings = nullptr,
+            const ASTPtr & query_ast_,
```

Comment:
Maybe better to delete `query_ast` at all? Mostly nullptr passed as a parameter, so it is useless.